### PR TITLE
Fix --suppress-permalink test

### PR DESCRIPTION
**Pull request by lukehoban**

### CODE REVIEW

File: tests/integration/integration_test.go [hunk 1]
```diff
@@ -461,7 +461,7 @@ func TestPermaLink(t *testing.T) {
 		Stdout:       stdout,
 		ExtraRuntimeValidation: func(t *testing.T, stackInfo integration.RuntimeValidationStackInfo) {
 			output := stdout.String()
-			assert.Contains(t, output, "PermaLink:")
+			assert.Contains(t, output, "Permalink:")
```
Comment (scott-the-programmer) [inline tests/integration/integration_test.go:464]: Sorry! Not sure how I missed this 😢

**Comment by lukehoban** [comment]
It appears the tests don't pass even with this fix - see https://travis-ci.com/github/pulumi/pulumi/builds/180080341.

I'll revert the PR and we can re-do it to fix this.

**Comment by scott-the-programmer** [comment]
Weird - I'll revisit my original PR later tonight 👍

**Comment by scott-the-programmer** [comment]
FYI - follow up PR here :) https://github.com/pulumi/pulumi/pull/5191

**Comment by stack72** [comment]
Closed via #5191

**Pull request closed without merging**